website/content/vagrant-cloud/hcp-vagrant/migration-guide.mdx

```diff
@@ -0,0 +1,149 @@
+
+##### HCP Projects
+
+[HCP projects](/hcp/docs/hcp/admin/orgs) allow organization owners to segment resources by team, environment, etc. Resources such as HCP Vagrant Registries, Hashicorp Virtual Private Networks (HPN). Each organization is assigned a `default-project` at creation and may have up to 10 projects.
+
+#### Create an HCP Vagrant Account
+
+To use the Vagrant Cloud migration tool you must define an organization and project within an HCP account and be signed into an active session on that account.
+
+Your HCP account must use the email address associated with your Vagrant Cloud account.Check under `Settings→ Profile` if you are unsure what email address you use for Vagrant Cloud.
+
+1. Sign up for a (free) [HCP account](https://portal.cloud.hashicorp.com/orgs/create), with your primary Vagrant Cloud organization email address. You must verify your account and sign in to HCP.
+2. At first log-in, use the wizard to create an HCP organization.
+3. Once you have created an organization you will be dropped into the dashboard for your `default-project`
+   - If you desire, you can create an additional project from the Projects Page.
+     - To get there, click `Projects` in the breadcrumbs at the top of the Dashboard page.
+     - A single organization may have up to 10 projects and each of those projects can have an unlimited number of [Registries](#registries) and Boxes.
+4. Now that you have a project in an HCP Account that uses the same email address as your Vagrant Cloud account, you’re ready to start migrating boxes!
+
+## Migrate to HCP Vagrant Registry
+
+#### 1. Sign in to both accounts and open the migration tool
+
+Once you have signed into both your HCP and Vagrant Cloud accounts, you can access the [migration tool](https://app.vagrantup.com/migration)
+
+To auto-populate the migration tool with a specific Vagrant Cloud `Organization` click on the `Migrate to HCP` button in the top right corner of your [settings](https://app.vagrantup.com/settings) page .
+
+#### 2. Select a Vagrant Cloud organization to migrate
+
+Select a single organization to migrate or select the `Migrate all organizations` checkbox to migrate them all at once to a single HCP Project.
+
+You may select any of the organizations listed in the Vagrant Cloud organization drop-down, even if a specific organization was pre-populated after navigating to the migration tool from your [Settings](https://app.vagrantup.com/settings) page.
+
+##### 3. Select a destination project in HCP
+
+If you have multiple projects in HCP, select the desired project from the HCP Project drop-down. If you have not created a second project the `default-project` will be auto-selected.
+
+When preparing for migration it is worth noting the following:
```

Comment:
I might move this entire section under the "## Migrate to HCP Vagrant Registry" header and change it to paragraph form.